R/misc.R

```diff
@@ -30,3 +30,182 @@
 "%not=%" <- function(a, b){
   ifelse(!is.na(a), a != b, F)
 }
+
+
+#' Check if a is greater than b
+#'
+#' @param a - a numeric variable present in the environment
+#' @param b - a numeric variable present in the environment
+#'
+#' @return - A boolean for the check
+#' @export
+#'
+#' @examples
+#' a <- 3
+#' a %_>_% 2
+"%_>_%" <- function(a, b){
+  ifelse(!is.na(a), as.numeric(a)>b, F)
+}
+
+
+#' Check if a is greater than or equal to b
+#'
+#' @param a - a numeric variable present in the environment
+#' @param b - a numeric variable present in the environment
+#'
+#' @return - A boolean for the check
+#' @export
+#'
+#' @examples
+#' a <- 5
+#' a %_>=_% 2
+"%_>=_%" <- function(a, b){
+  ifelse(!is.na(a), as.numeric(a)>=b, F)
+}
+
+
+#' Check if a is less than b
+#'
+#' @param a - a numeric variable present in the environment
+#' @param b - a numeric variable present in the environment
+#'
+#' @return - A boolean for the check
+#' @export
+#'
+#' @examples
+#' a <- 5
+#' a %_<_% 6
+"%_<_%" <- function(a, b){
+  ifelse(!is.na(a), as.numeric(a)<b, F)
+}
+
+
+#' Check if a is less than or equal to b
+#'
+#' @param a - a numeric variable present in the environment
+#' @param b - a numeric variable present in the environment
+#'
+#' @return - A boolean for the check
+#' @export
+#'
+#' @examples
+#' a <- 5
+#' a %_<=_% 7
+"%_<=_%" <- function(a, b){
+  ifelse(!is.na(a), as.numeric(a)<=b, F)
+}
+
+
+#' Sum of two numerics
+#'
+#' @param a - a numeric variable present in the environment
+#' @param b - a numeric variable present in the environment
+#'
+#' @return  - A numeric sum
+#' @export
+#'
+#' @examples
+#' a <- 4
+#' a %_+_% 2
+"%_+_%" <- function(a, b){
+  as.numeric(a) + as.numeric(b)
+}
+
+#' Compare two variables are the same and are not NA
+#'
+#' @param e1 - a variable present in the environment
+#' @param e2 - a variable present in the environment
+#'
+#' @return - a boolean result of the comparison
+#' @export
+#'
+#' @examples
+#' a <- 4
+#' b <- 5
+#' a %==na% b
+"%==na%" <- function(e1, e2){
+  ifelse(is.na(e1 == e2), is.na(e1) == is.na(e2), e1 == e2)
+}
+
+
+
+
+#' Check variables are non-equal and not NA
+#'
+#' @param e1 - a variable present in the environment
+#' @param e2 - a variable present in the environment
+#'
+#' @return - a boolean result of the comparison
+#' @export
+#'
+#' @examples
+#' a <- 4
+#' b <- 5
+#' a %not=na% b
+"%not=na%" <- function(e1, e2){
+  (e1 != e2 | (is.na(e1) & !is.na(e2)) | (is.na(e2) & !is.na(e1))) & !(is.na(e1) & is.na(e2))
+}
+
+#' A shorthand is NA function
+#'
+#' @param x - a variable present in the environment
+#'
+#' @return - a boolean of whether X was NA
+#' @export
+#'
+#' @examples
+#' a <- 2
+#' isna(2)
+isna <- function(x){
+  is.na(x)
```

Comment:
Can you try to add a test to isna?
The CodeCov is being affected by not having one.